jest.config.common.js

```diff
@@ -1,9 +1,6 @@
 module.exports = {
   transform: {
-    '.(ts|tsx)': [
-      'ts-jest',
-      { tsconfig: { downlevelIteration: true, esModuleInterop: true, strictNullChecks: false } },
```

Comment:
>downlevelIteration [](http://example.com/codeflow?start=20&length=18)

We don't need the `downlevelIteration` flag anymore because we are now running against ES6 which has the iteration behavior we were previously pushing downlevel into ES5 with this flag